docs/man_pages/device/device-log.md

```diff
@@ -1,24 +1,36 @@
 device log
 ==========
 
-Usage | Syntax
+Usage | Synopsis
 ------|-------
-General | `$ appbuilder device log --device <Device ID>`
-Opens the device log stream for a selected connected device. 
+General | `$ appbuilder device log [--device <Device ID>]`
+
+Opens the device log stream for a selected connected Android <% if(isWindows || isMacOS) { %>or iOS <% } %>device. 
+
+<% if(isConsole) { %>
+WARNING: You can work only with connected <% if(isWindows || isMacOS) { %>iOS and <% } %>Android devices.  
+<% if(isLinux) { %>WARNING: You cannot work with connected iOS devices on Linux systems. To view the complete help for this command, run `$ appbuilder help device log`<% } %> 
```

Comment:
On linux I receive two warnings:
```
WARNING: You can work only with connected Android devices.
WARNING: You cannot work with connected iOS devices on Linux systems. To view the complete help for this command, run $ appbuilder help device list-applications
```

I think the oce for iOS devices is redundant and can be removed.